### DISCUSSION TITLE
How to get GMT time by using time.format

### DISCUSSION THREAD

**Discussion by JinXiu2023** [open-policy-agent/community#430] [OPA and Rego] [answered]
hello, I want to get a RFC822/RFC1123 time  and the time zone is GMT standard time like `"Tue, 23 May 2023 08:54:15 GMT"`. but I got wrong response using time.format. I need your help. And thank you anyway.

This is the code.
```go
package example

date1 := time.format(time.now_ns())
date2 := time.format([time.now_ns(), "Asia/Shanghai"])
date3 := time.format([time.now_ns(), "", "RFC1123"])
date4 := time.format([time.now_ns(), "", "%a,%d%b%Y %H:%M:%S GMT"])
```

This is the answer.
```go
{
    "date1": "2023-05-24T03:50:39.596970532Z",
    "date2": "2023-05-24T11:50:39.596970532+08:00",
    "date3": "RFC55243",
    "date4": "%a,%d%b%Y %H:%M:%S GMT"
}
```

**Comment by anderseknert** [comment, marked as answer]
Good morning! Yeah, ideally you'd be able to use some of the predefined layouts for formatting, but for now you'll have to use the reference format as defined in Golang, i.e:

```
t := time.format([time.now_ns(), "GMT", "Mon, 02 Jan 2006 15:04:05 MST"])
```

See https://gosamples.dev/date-time-format-cheatsheet/ for some alternatives.

**Reply by JinXiu2023** [reply]
Wonderful! Thank you.
